tests/import_specifications/test_individual_parsers.py

```diff
@@ -855,3 +874,57 @@ def test_dts_manifest_file_is_directory(temp_dir: Generator[Path, None, None]):
             )
         ],
     )
+
+
+@pytest.mark.parametrize("bad_schema", [None, 1, [], {"foo"}])
+def test_dts_manifest_bad_schema(bad_schema):
+    f = _get_test_file("manifest_small.json")
+    with pytest.raises(Exception):
+        parse_dts_manifest(f, bad_schema)
```

Comment:
No checking the error message?

Reply:
Moot with the change to pass in the validator.

Reply:
Done.

Reply:
:+1: